tests/test_enum.rs

```diff
@@ -0,0 +1,50 @@
+use pyo3::prelude::*;
+use pyo3::{py_run, wrap_pyfunction};
+
+mod common;
+
+#[pyenum]
+#[derive(Debug, PartialEq, Clone)]
+pub enum MyEnum {
+    Variant = 1,
+    OtherVariant = 2,
+}
+
+#[test]
+fn test_reflexive() {
+    let gil = Python::acquire_gil();
+    let py = gil.python();
+    let mynum = py.get_type::<MyEnum>();
+    py_assert!(py, mynum, "mynum.Variant == mynum.Variant");
+    py_assert!(py, mynum, "mynum.OtherVariant == mynum.OtherVariant");
+}
+
+#[pyfunction]
+fn return_enum() -> MyEnum {
+    MyEnum::Variant
+}
+
+#[test]
+fn test_return_enum() {
+    let gil = Python::acquire_gil();
+    let py = gil.python();
+    let f = wrap_pyfunction!(return_enum)(py);
+    let mynum = py.get_type::<MyEnum>();
+
+    py_run!(py, f mynum, "assert f() == mynum.Variant")
+}
+
+#[pyfunction]
+fn enum_arg(e: MyEnum) {
+    assert_eq!(MyEnum::OtherVariant, e)
+}
+
+#[test]
+fn test_enum_arg() {
+    let gil = Python::acquire_gil();
+    let py = gil.python();
+    let f = wrap_pyfunction!(enum_arg)(py);
+    let mynum = py.get_type::<MyEnum>();
+
+    py_run!(py, f mynum, "f(mynum.Variant)")
```

Comment:
If you take my suggestion from above, we can test both cases:

```suggestion
    py_run!(py, f mynum, "assert f(mynum.Variant) == 'Variant'")
    py_run!(py, f mynum, "assert f(mynum.OtherVariant) == 'OtherVariant'")
```